src/UI/class-settings-page.php

```diff
@@ -303,11 +303,10 @@ private function initialize_basic_section(): void {
 		);
 
 		// Site ID.
-		$h          = __( 'Your Site ID is typically your own site domain without <code>http(s)://</code> prefixes or trailing <code>/</code> (e.g. <code>mydomain.com</code>).', 'wp-parsely' );
```

Comment:
Used the help text directly and avoid the declaration of `$h` variable. To me it seems easy to read when every info which is only used once is available in the `$field_args`

Reply:
I agree.